### PR TITLE
Update dependencies slightly, support .zsh_history with no timestamps

### PR DESCRIPTION
**Pull request by anthonyheckmann**
I had trouble getting this to run on my M1 macbook, and a newer version of Node, so I used yarn to update the sql library and type definitions. 

Tested w/ node 18.5.0 and yarn 1.18. Maybe that should be in the packages.json too.

### CODE REVIEW

File: insert-all.ts [hunk 1]
```diff
@@ -3,7 +3,6 @@ import Sqlite from "better-sqlite3"
 import { readFileSync } from "fs"
 import { hostname } from "os"
 import { join } from "path"
-import { createInterface } from "readline"
```
Comment (anthonyheckmann) [inline insert-all.ts:6]: seemed unused

File: insert-all.ts [hunk 2]
```diff
@@ -57,19 +58,23 @@ async function* readEntries() {
 			continue
 		}
 
-		const history_entry_regex = /^: (?<started>\d+):(?<duration>\d+);(?<command>[\s\S]*)$/
+		const history_entry_regex = /^(: (?<started>\d+):(?<duration>\d+);)?(?<command>[\s\S]*)$/
```
Comment (anthonyheckmann) [inline insert-all.ts:61]: makes the first two capture groups optional

File: insert-all.ts [hunk 3]
```diff
@@ -57,19 +58,23 @@ async function* readEntries() {
...
 			throw Error(
-				`invalid history syntax on line ${line} in ${historyFile}: \n"${entry}"`,
+				`no history syntax and no command on line ${line} in ${historyFile}: \n"${entry}"`,
 			)
 		}
-
-		yield result.groups as Entry
+		const groups = result.groups
+		if (groups) {
+			const { started = DEFAULT_DATE, duration = DEFAULT_DURATION, command } = groups;
```
Comment (anthonyheckmann) [inline insert-all.ts]: set default values for any missing members of Entry

**Comment by phiresky** [comment]
thanks!

**Pull request merged**